homeassistant/components/ezbeq/entity.py

```diff
@@ -0,0 +1,31 @@
+"""Base class for ezbeq entities."""
+
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import EzBEQCoordinator
+
+
+class EzBEQEntity(CoordinatorEntity[EzBEQCoordinator]):
+    """Defines a base ezbeq entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(self, coordinator: EzBEQCoordinator, device_name: str) -> None:
+        """Initialize ezbeq entity."""
+        super().__init__(coordinator)
+        assert coordinator.config_entry
+
+        self._attr_device_info = DeviceInfo(
+            identifiers={
+                (DOMAIN, f"{coordinator.config_entry.entry_id}_{device_name}")
```

Comment:
Do we have something more unique for the device other than the device name?

Reply:
no it doesnt expose any more identifying info than the name

Reply:
Can the device name change?

Reply:
theoretically the user could change the name of the physical device